app/components/tables/headers/sort.js

```diff
@@ -15,6 +15,12 @@ export default class Sort extends Component {
         return 'caret down';
       }
     }
+    else {
+      //default sorting for starts-at column
```

Comment:
Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)